demo/test/utill/a11y.js

```diff
@@ -0,0 +1,39 @@
+const pa11y = require('pa11y');
+
+module.exports = (testPage, method = 'GET') => {
+  const [host] = /(^http.+:\d+\/)/.exec(testPage);
+  const indexUrl = `${host}index`;
+  console.log("testPage::::::"+testPage)
```

Comment:
can use , instead of + in console.log

console.log("testPage::::::", testPage) 

(or)

console.log(`testPage is... ${testPage}`)